tests/integration/podman/Containerfile_archlinux

```diff
@@ -5,3 +5,11 @@ RUN pacman -Syu --noconfirm git
 COPY ./clone_and_install_kw.sh .
 
 RUN bash ./clone_and_install_kw.sh
+
+COPY ./scripts /scripts
```

Comment:
I'm confused about this part; where does this `scripts` folder come from? Is this the script folder from kw that has `kwreview.sh  profiler.sh`? If so, why do you need it?

Reply:
No, this `/scripts` is from the path `tests/integration/podman/scripts`